js/lib/cli.js

```diff
@@ -326,6 +349,12 @@ function processInputSync(filepath) {
 
 function makePretty(code, config, outfile, callback) {
     try {
+        if (config.editorConfig) {
```

Comment:
I was trying to do this but I had some difficulties. This is because you have to obtain the editorconfig settings for each individual file, that is after the parsing of all the configs but before the prettifier actually starts to work. It is hard to override the settings at that time since there is no way to detect if the configuration is set by a default value.